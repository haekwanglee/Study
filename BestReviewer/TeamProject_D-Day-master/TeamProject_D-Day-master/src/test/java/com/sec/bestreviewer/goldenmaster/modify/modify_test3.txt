SCH,-p,-f, ,name,FIRST
MOD, , , ,employeeNum,90000000,name,Modified Name
SCH,-p, , ,name,Modified Name
SCH,-p,-f, ,name,Modified
SCH,-p,-l, ,phoneNum,1111
