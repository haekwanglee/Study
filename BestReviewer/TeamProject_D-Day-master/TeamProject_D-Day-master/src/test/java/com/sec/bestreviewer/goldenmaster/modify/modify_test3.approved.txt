SCH,90000000,FIRST MAN,CL2,010-1111-1111,19980101,ADV
MOD,1
SCH,90000000,Modified Name,CL2,010-1111-1111,19980101,ADV
SCH,90000000,Modified Name,CL2,010-1111-1111,19980101,ADV
SCH,90000000,Modified Name,CL2,010-1111-1111,19980101,ADV
SCH,95000002,BBBBB BBB,CL2,010-2222-1111,19980201,ADV
SCH,00000001,DICTIONARY A,CL2,010-3333-1111,19980301,ADV
SCH,20000003,DICTIONARY ACC,CL2,010-4444-1111,19980801,ADV
SCH,33000001,CERTIADV BABY,CL2,010-5555-1111,20000701,ADV
